### PR TITLE
chore(gulp): changes to gulpfile to run on windows

### PR DESCRIPTION
**Pull request by cnishina**

### CODE REVIEW

**Comment by Andre-H** [comment]
Thank you very much for this @cnishina. Worked on our Windows 7 box. I did had to alter website/package.json though, or else it wouldn't install on Windows.

`"scripts": {
    "prepublish": "node node_modules/bower/bin/bower install",`

**Comment by cnishina** [comment]
I will get that into this PR... Thanks for catching that.

File: gulpfile.js [hunk 1]
```diff
@@ -20,11 +31,11 @@ gulp.task('built:copy', function() {
...
 
 gulp.task('webdriver:update', function(done) {
-  runSpawn(done, 'bin/webdriver-manager', ['update']);
+  runSpawn(done, 'node', ['bin/webdriver-manager', 'update']);
 });
 
-gulp.task('jslint', function(done) {
-  runSpawn(done, './node_modules/.bin/jshint', ['lib','spec', 'scripts']);
+gulp.task('jshint', function(done) {
+  runSpawn(done, 'node', ['node_modules/jshint/bin/jshint', 'lib', 'spec', 'scripts']);
```
Comment (juliemr) [inline gulpfile.js]: Should this be `jshint.js`?

Comment (juliemr) [inline gulpfile.js]: Answering my own question, no, it's good the way it is.

**Pull request merged**